tools/gke-disk-image-builder/script/startup.sh

```diff
@@ -17,6 +17,14 @@ sudo apt-get update
 
 # Install containerd.
 sudo apt install --yes containerd
+# Configure docker.io registry mirror
+sudo mkdir -p /etc/containerd/certs.d/docker.io
+sudo tee /etc/containerd/certs.d/docker.io/hosts.toml <<EOF
```

Comment:
Do you not need to check for the existence of this file before creating it to avoid overwriting it?